packages/database/test/query.test.ts

```diff
@@ -2191,6 +2191,49 @@ describe('Query Tests', () => {
       });
   });
 
+  it('get() at empty root returns null', done => {
```

Comment:
@schmidt-sebastian Can you take a look at these tests? We discussed specing out the behavior of get() as it relates to listeners, but I don't think get() should interact with them directly in the JS sdk since they should be up to date anyway.

Reply:
Got it. I think the main reason I asked for these tests is to make sure that values returned via `get` are cached. What you can do is maybe follow the pattern in https://github.com/firebase/firebase-js-sdk/blob/55f09a2642043d9fe9415c069e1dc82e1bcb612c/packages/database/test/info.test.ts#L142 and create two database instances. The first instance is just for writing a new value, the second value would call `get()` followed by `once()` to ensure that `once()` returns the correct value.